install_config/aggregate_logging.adoc

```diff
@@ -910,65 +898,16 @@ As part of the OpenShift cluster installation, it is recommended that you add th
 
-Fluentd uses `journald` as the system log source. These are log
-messages from the operating system, Docker, and OpenShift. For
-container logs, Fluentd determines which log driver Docker is using,
-`json-file` or `journald`, and automatically reads the logs from that source.
-
-// [[fluentd-use-journald]]
-// *Having Fluentd Use the Systemd Journal as the Log Source*
-
-// By default, Fluentd reads from `journald` and
-// *_/var/log/containers/<container>.log_* for system logs and container logs,
-// respectively. You can instead use the systemd journal as the log source. There
-// are three inventory parameters available:
-
-// [cols="3,7",options="header"]
-// |===
-// |Parameter
-// |Description
-
-// | `journal-source`
-// |The default is empty, so that when using the systemd journal, Fluentd first looks for
-// *_/var/log/journal_*, and if that is not available, uses *_/run/log/journal_*
-// as the journal source. You can specify `journal-source` with an explicit
-// journal path. For example, if you want Fluentd to always read logs
-// from the transient in-memory journal, set `journal-source`=*_/run/log/journal_*.
-
-// | `openshift_logging_journal_read_from_head`
-// |If this setting is `false`, Fluentd starts reading from the end of the journal,
-// ignoring historical logs. If this setting is `true`, Fluentd starts reading
-// logs from the beginning of the journal.
-// |===
-
-[NOTE]
-====
-As of {product-title} 3.3, Fluentd no longer reads historical log files when
-using the JSON file log driver. In situations where clusters have a large number
-of log files and are older than the EFK deployment, this avoids delays when
-pushing the most recent logs into Elasticsearch. Curator deleting logs are
-migrated soon after they are added to Elasticsearch.
-====
+With Fluentd, it is recommended to use `json-file` as the system log source.
+These are log messages from the operating system, Docker, and OpenShift. For
+container logs, Fluentd determines which log driver Docker is using, `json-file`
+or `journald`, and automatically reads the logs from that source.
 
 [NOTE]
 ====
-It may require several minutes, or hours, depending on the size of your
-journal, before any new log entries are available in Elasticsearch, when using
-`openshift_logging_journal_read_from_head=true`.
-====
-
-[WARNING]
-====
-It is highly recommended that you use the default value for `use-journal`. In
-scenarios where upgrading {product-title} changes the Docker log driver, if
-`use-journal=False` is explicitly specified as part of installation, Fluentd
-still expects to read logs generated using the `json-file` log driver. This
-results in a lack of log ingestion. If this has happened within your logging cluster,
-xref:fluentd-upgrade-source[troubleshoot] it.
-
-See
-xref:fluentd-upgrade-source[Updating Fluentd's Log Source After a Docker Log
-Driver Update] for more information.
+As of {product-title} 3.9, support for `journald` is being deprecated. It is
```

Comment:
@portante @richm @tkatarki I don't think this is EXACTLY correct.  We still support journal right.  The caveat is:

* We ALWAYS gather node logs from journal
* App logs CAN come from journal but there are limitations.  We recommand json

Reply:
We are deprecating support for the docker `journald` log driver, we are not deprecating support for `journald` itself, as we always collect operational logs from it.  I don't think we should continue to support and verify that we can extract container logs from `journald`.

Reply:
@portante Then we need to officially deprecate it, and go through the official deprecation procedure.

Reply:
I removed the line about deprecating support for the `journald` log driver here, since there seemed to be some value in it from the email conversation. I am leaving the notes and content around suggesting the use of the `json-file` log driver.